### ISSUE TITLE
If statement missing in scrollbar

### ISSUE THREAD

**Issue by bomtvedt4** [TomSchimansky/CustomTkinter#2290]
Hello,

I have been trying to implement a scroll bar into one of my projects but it kept returning this error. "ValueError: ['orientation'] are not supported arguments. Look at the documentation for supported arguments."

Looks like in the ctk_scrollbar function "def configure(self, require_redraw=False, **kwargs), 'orientation' is not listed in the if statements.

Thanks

**Comment by AshhadDevLab** [comment]
That's true the configure function in the scrollbar doesn't support the changing of the scroll direction, by default the scroll direction is set to vertical but it can be changed only at the time of the creation of the scrollbar and not after creating.
